### PR TITLE
Migrating helpcenter content

### PR DESCRIPTION
**Pull request by mcclowes**

### CODE REVIEW

File: docs/configure/integrations.md [hunk 1]
```diff
@@ -20,7 +18,25 @@ Get up and running with Xero quickly to test a real integration by following our
 
 You can enable the integrations you need [via the Portal](https://app.codat.io/settings). Each integration configuration page shows the needed credentials, and includes links to integration-specific documentation and the indegration's own developer portal.
 
-See our detailed integration-specific [documentation](/integrations/all-integrations) for more informatino about the integrations you need.
+See our detailed integration-specific [documentation](/integrations/all-integrations) for more information about the integrations you need.
+
+### Accounting Data
+
+Using the Codat Portal, you have the option to choose which accounting packages should be shown on the link site. For detailed guides please see
```
Comment (dcoplowe) [inline docs/configure/integrations.md:25]: See what?

File: docs/auth-flow/build/faqs.md [hunk 2]
```diff
@@ -3,10 +3,24 @@ title: "Auth flow FAQs"
 slug: "faqs"
 excerpt: "Hints, tips and answers on improving your bespoke auth flow experience"
 hidden: false
-createdAt: "2022-11-22T16:34:56.484Z"
-updatedAt: "2022-11-23T15:15:10.291Z"
 ---
 
+## Best Practices
+
+If an end user has linked before then the same company id should be used always, even if they deauthorize.
```
Comment (dcoplowe) [inline docs/auth-flow/build/faqs.md:10]: I don't really understand what this sentence is trying to say. Are we saying that you should only create a single company per SMB and manage the associated connections accordingly. For example, if a company once had a `Linked` connection that was later deauthorised you can manage re-authorising the data connection using the same company Id and connection Id?

File: docs/get-started/support.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Support"
+---
+
+Codat provides a second line support function during UK business hours (09:00 to 17:00 Monday to Friday) and can be contacted by emailing support@codat.io.
+
+:::caution Access to support
+
+If you are on our Start up plan, please note this does not include access to support.
```
Comment (dcoplowe) [inline docs/get-started/support.md:9]: Any chance we can say what support is offered/link out to a place where users can see the Start up plan support?

File: docs/get-started/support.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+This can be accessed here https://status.codat.io/
+
+## Raising a ticket
+
+If you run into an issue and need to raise a support ticket, you can do so by emailing support@codat.io.
+
+To help us deal with your query efficiently please include the following:
+
+- Company ID
+- Unique Codat Company Id (unique 36-character identified)
```
Comment (dcoplowe) [inline docs/get-started/support.md:29]: Duplication

File: docs/zendesk-content/additional-sync-settings.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+
+
+Configuration
+
+Configuration is required per data type. The current settings, retrieved using GET, can be used to develop the body of each POST request below.
+
+All Companies
+
+POST /profile/syncSettings (Swagger)
```
Comment (dcoplowe) [inline docs/zendesk-content/additional-sync-settings.md:62]:
```suggestion
POST /profile/syncSettings
```

**Review by dcoplowe — APPROVED** [review]

**Comment by pzaichkina** [comment]
I would LOVE to spend some time on the actual text of these. what is the plan? many of them don't seem to be referenced in the menus yet - I guess keep these hidden while we edit them?

File: docs/auth-flow/build/faqs.md [hunk 6]
```diff
@@ -3,10 +3,24 @@ title: "Auth flow FAQs"
...
+Unless you are 100% sure you want to delete all history of things having happened in which case delete the company and remove the association.
```
Comment (mcclowes) [inline docs/auth-flow/build/faqs.md:11]:
```suggestion
```

File: docs/configure/integrations.md (hunk 1 above)
Comment (mcclowes) [inline docs/configure/integrations.md:25]:
```suggestion
Using the Codat Portal, you have the option to choose which accounting packages should be shown on the link site.
```

File: docs/get-started/support.md (hunk 3 above)
Comment (mcclowes) [reply docs/get-started/support.md:9]: This page isn't shown yet

**Pull request merged**